cmd/buildah/main.go

```diff
@@ -26,6 +27,12 @@ func main() {
 		cli.OsExiter(1)
 	}
 
+	err = utils.CheckConfigFiles()
+	if err != nil {
+		logrus.Errorf(err.Error())
+		cli.OsExiter(1)
```

Comment:
Just as a test, could you comment out this line temporarily and push a new patch?  I'd like to see if the tests are happy that way.  Then we can figure out if we should change this to a warning or figure out a way to turn off this check during testing.

Reply:
ok let's see if can pass :eyes: